bool siloInputDoor
int8 siloConcreteLevel1
int8 siloConcreteLevel2
bool siloBreaker
bool siloDoser
bool siloHopperLevel
bool siloTrasmit
int8 towerLifting
bool towerLiftError
bool towerFunnelBreaker
bool towerFunnelBreakerError
bool towerAugerMotor1
int8 towerAugerMotor1Speed
int8 towerAugerMotor1Current
int8 towerAugerMotor1Torque
bool towerAugerMotor2
int8 towerAugerMotor2Speed
int8 towerAugerMotor2Current
int8 towerAugerMotor2Torque
